metrics.go

```diff
@@ -17,13 +17,18 @@ import (
 type MetricType string
 
 const (
-	builtinMetricsPrefix = "juju"
+	BuiltinMetricPrefix = "juju"
 
 	// Supported metric types.
 	MetricTypeGauge    MetricType = "gauge"
 	MetricTypeAbsolute MetricType = "absolute"
 )
 
+// IsBuiltinMetric returns true if the metric key provided is in the builtin metric namespace.
```

Comment:
But tbh I find it hard to get excited about using this rather than just exporting BuiltinMetricPrefix.